lib/libzrepl/data_conn.c

```diff
@@ -361,6 +362,19 @@ uzfs_zvol_worker(void *arg)
 			atomic_inc_64(&zinfo->sync_req_received_cnt);
 			break;
 
+		case ZVOL_OPCODE_UNMAP:
+			if ((rebuild_cmd_req &&
```

Comment:
why only for rebuild_cmd_req?

Reply:
it is for both.. for rebuild and also for app IOs